toolchains/ci_workspaces/WORKSPACE.x86_64.release.rhel.tmpl

```diff
@@ -58,14 +58,14 @@ http_archive(
     name = "libtorch",
     build_file = "@//third_party/libtorch:BUILD",
     strip_prefix = "libtorch",
-    urls = ["https://download.pytorch.org/libtorch/nightly/cu124/libtorch-cxx11-abi-shared-with-deps-latest.zip"],
+    urls = ["https://download.pytorch.org/libtorch/test/cu124/libtorch-cxx11-abi-shared-with-deps-2.4.0%2Bcu124.zip"],
```

Comment:
Do the cu124 terms need to be templatized so it gets filled in with the prebuild script?